pkg/arvo/sys/hoon.hoon

```diff
@@ -11534,114 +11557,175 @@
     ::
-    ::  step: indent
-    ::  into: :: and indent to end of line, consuming following space.
-    ::  indo: :: to end of line, consuming following space.
-    ::  exit: :: to end of line, not consuming following space.
+    ++  rant
+      |*  sec=rule
+      %-  star
+      ;~  pfix
+        (into null)
+        (plus (into sec))
+      ==
     ::
+    ++  null  (cold ~ (star ace))
+    ++  text  (pick line code)
+    ++  teyt  (pick line ;~(pfix step code))
+    ++  line  ;~(less ace (cook crip (star prn)))
+    ++  code  ;~(pfix step ;~(less ace (cook crip (star prn))))
     ++  step  ;~(plug ace ace)
-    ++  into  |*(bod=rule (indo ;~(pfix step bod)))
     ::
-    ++  indo
+    ++  into
       |*  bod=rule
-      ;~(pfix col gar ;~(sfix bod (just `@`10) (punt gap)))
+      (ifix [;~(plug col col) ;~(plug (just '\0a') (star ace))] bod)
     ::
     ++  exit
       |*  bod=rule
-      ;~(pfix (star ace) col gal step bod)
-    ::
-    ::  fill: full definition
+      ;~(pfix (star ace) col col step bod)
     ::
-    ++  fill
-      %+  cook  |=([[a=term b=cord] c=(list sect) (unit ~)] [a b c])
-      ;~  plug
-        (into fine)
-        (rant ;~(pfix step text))
-        (punt (indo null))
-      ==
-    ::
-    ::  rant: series of sections.
-    ::
-    ++  rant
-      |*  sec=rule
-      %-  star
-      ;~  pfix
-        (indo null)
-        (plus (into sec))
+    ++  en-link
+      |=  a=nail  %.  a
+      %+  knee  *link  |.  ~+
+      %-  stew
+      ^.  stet  ^.  limo
+      :~  :-  '|'
+          ;~(pfix bar (stag %chat sym))
+          :-  '.'
+          ;~(pfix dot (stag %frag sym))
+          :-  '+'
+          ;~(pfix lus (stag %funk sym))
+          :-  '$'
+          ;~(pfix buc (stag %plan sym))
+          :-  '%'
+          ;~(pfix cen (stag %cone bisk:so))
+          :-  '^'
+          ;~(pfix ket (stag %core sym))
+          :-  '_'
+          ;~(pfix cab (stag %door sym))
+          :-  '='
+          ;~(pfix tis (stag %gate sym))
+          :-  '/'
+          ;~(pfix fas (stag %path sym))
       ==
     --
   ::
+  ++  clad                                              ::  hoon doccords
+    |*  fel=rule
+    %+  cook
+      |=  [a=whit b=hoon c=help]
+      =?  b  !=(*help c)
+        [%note help+c b]
+      =+  docs=~(tap by bat.a)
+      |-
+      ?~  docs  b
+      $(docs t.docs, b [%note help+[i.docs] b])
+    (seam fel)
+  ++  coat                                              ::  spec doccords
+    |*  fel=rule
+    %+  cook
+      |=  [a=whit b=spec c=help]
+      =?  b  !=(*help c)
+        [%gist c b]
+      =+  docs=~(tap by bat.a)
+      |-
+      ?~  docs  b
+      $(docs t.docs, b [%gist i.docs b])
+    (seam fel)
+  ++  scye                                              ::  prefix doccords only
+    |*  fel=rule
+    ;~(pose ;~(plug apex:docs ;~(pfix gap fel)) ;~(plug (easy *whit) fel))
+  ++  seam                                              ::  postfix doccords too
```

Comment:
```suggestion
  ++  seam                                              ::  doccords
```

Alternatively, "with doccords" here, and "with prefix docs" or similar for `+scye`.